src/Compilers/CSharp/Test/Semantic/Semantics/OperatorTests.cs

```diff
@@ -51,10 +51,10 @@ class C {
 
             Assert.NotEqual(insideEnumDefinition, insideMethodBody);
 
-            Assert.Equal(insideEnumDefinition.Symbol.ToTestDisplayString(),
-                "System.Int32 System.Int32.op_BitwiseOr(System.Int32 left, System.Int32 right)");
-            Assert.Equal(insideMethodBody.Symbol.ToTestDisplayString(),
-                "TestEnum TestEnum.op_BitwiseOr(TestEnum left, TestEnum right)");
+            Assert.Equal(                "System.Int32 System.Int32.op_BitwiseOr(System.Int32 left, System.Int32 right)",
+insideEnumDefinition.Symbol.ToTestDisplayString());
+            Assert.Equal(                "TestEnum TestEnum.op_BitwiseOr(TestEnum left, TestEnum right)",
+insideMethodBody.Symbol.ToTestDisplayString());
```

Comment:
Formatting looks odd.

Reply:
@jaredpar This one might be worth filing a bug on https://github.com/xunit/xunit

Reply:
nit: Re-activated, as the formatting still looks odd

---
In reply to: [327207409](https://github.com/dotnet/roslyn/pull/38789#discussion_r327207409) [](ancestors = 327207409)